### DISCUSSION TITLE
Know if player is muted

### DISCUSSION THREAD

**Discussion by MayureshKarekarMG** [PierfrancescoSoffritti/android-youtube-player#1034] [General]
Is there a way to know if the player is muted or unmuted?

**Comment by PierfrancescoSoffritti** [comment]
You mute/unmute programmatically. Is that what you're asking?

**Reply by MayureshKarekarMG** [reply]
No, I want to know if the player volume is muted currently or current volume level.

**Reply by MayureshKarekarMG** [reply]
@PierfrancescoSoffritti Any updates on this? I want to know the player's audio muted status. Something like 
`val isVideoMuted = youTubePlayer.isMuted`

**Reply by PierfrancescoSoffritti** [reply]
This is currently not possible. Although [the player has a method for it](https://developers.google.com/youtube/iframe_api_reference#changing-the-player-volume). The library currently has no way to call a method in javascript and get a result back, but I think it should be doable.

**Reply by MayureshKarekarMG** [reply]
Understood @PierfrancescoSoffritti , Thanks for the quick support.

**Reply by PierfrancescoSoffritti** [reply]
I have opened #1045 to keep track of this.
